### DISCUSSION TITLE
使用代理集时，如何制定某网站使用代理集中的某节点？

### DISCUSSION THREAD

**Discussion by GeekXtop** [vernesong/OpenClash#4248] [Q&A]
当使用配置订阅模式时
可以在覆写设置-规则设置中追加某网站规则，如：
`- DOMAIN-SUFFIX,quins.us,🇹🇭 Thailand | 01`
而当添加代理集自动生成配置文件时，经过尝试只能使用策略组名称，如：
`- DOMAIN-SUFFIX,quins.us,Asian TV`
具体到节点名称该规则会导致内核崩溃

提问：
使用代理集时，如何制定某网站使用代理集中的某节点？

**Comment by fejeIaqbhbv** [comment]
我建议去内核那边问吧
那边思路广的人多

代理集加载顺序晚于运行配置，所以没有直接方案
硬要说的话，专门写一个 `proxy-group` 然后 `filter` 限定到节点上？
